# HG changeset patch
# Parent  598584f3f664456ec33a7a8baac2d1dc3be3b1b0
Add a release command to generate Docker images

diff --git a/dist/docker/Dockerfile b/dist/docker/Dockerfile
new file mode 100644
--- /dev/null
+++ b/dist/docker/Dockerfile
@@ -0,0 +1,15 @@
+FROM lopter/openwrt_designated_driver_{{ isa }}
+MAINTAINER Louis Opter <louis@opter.org>
+
+ADD https://downloads.lightsd.io/openwrt/designated_driver_trunk/{{ isa }}/lightsd_{{ version }}_{{ isa }}.ipk /tmp/
+
+RUN opkg update \
+    && opkg install \
+        libevent2-core \
+        /tmp/lightsd_{{ version }}_{{ isa }}.ipk \
+    && rm -rf /tmp/*
+
+# TODO: figure out user creation and networking
+
+USER lightsd
+ENTRYPOINT /usr/bin/lightsd -v warning -t -s /var/run/lightsd/socket -c /var/run/lightsd/pipe
